modules/rosa-enable-private-cluster-new.adoc

```diff
@@ -0,0 +1,29 @@
+:_module-type: PROCEDURE
+// Module included in the following assemblies:
+//
+// rosa_aws_private_connections/rosa-private-cluster.adoc
+
+
+[id="rosa-enabling-private-cluster-new_{context}"]
+= Enabling private cluster on a new cluster
+
+[role="_abstract"]
+You can enable the private cluster setting when creating a new {product-title} cluster.
+
+.Prerequisites
+
+AWS VPC Peering, VPN, DirectConnect, or TransitGateway has been configured to allow private access.
```

Comment:
same thought here, can this be highlighted somehow?

Reply:
Same comment as above.